README.md

```diff
@@ -37,33 +37,69 @@ documentation](https://spladder.readthedocs.io/en/latest/).
+- `--output-fasta`: Default = False. Set this parameter to True to output the foreground peptides in fasta format. If set to True output number 4 from the [Output files, mode build](#outputs-mode-build) will be generated. If set to False only the kmers will be outputted.
+- `--force-ref-peptides`: Default = False. Set this parameter to True to output mutated peptides even if they are the same as the ones in the reference.
+- `--kmer-database`: Default = None. Absolute path of a file containing kmers in one column, without header. If the kmers contained in this database contain the aminoacid isoleucine (I), it will be converted into leucine (L). A file from uniprot or any other standard library can be provided. The kmers provided in this file will not be outputted if found in the foreground peptides. Please note that is a standard proteome is downloaded from an online resource the proteins should be cut into the kmers length selected under `--kmer`.
+- `--gtex-junction-path`: Default = None. Absolute path of whitelist junction path. The junctions of this file will be the only ones outputted from the tool. Format: hdf5. The hdf5 file should have a key `chrms` indicating the chromosome where the junction is located, a key `pos` containing the starting and ending position of the junction in genomic coordinates, and a key `strand` with information on whether the junction is in the '+' or '-' strand.
+- `--disable-concat`: Default = False. Disable the generation of kmers from combinations of more than 2 exons. In this mode, any kmer shorter than `--kmer` will be discarded. By setting this command to False, the generation of kmers from 3 exons is allowed. This might ensure that kmers generated from shorter exons are kept, but one should take into account that kmers translated from 3 exons might have lower confidence. By setting the argument to True the generation of kmers is faster.
+- `--disable-process-libsize`: Default = False. Set to True to generate the libsize file (file 7 from the [output section, mode build](#output-mode-build)).
+
+*Optional mutation* parameters: Commands to add somatic and germline mutation information.
+- `--mutation-sample`: Default = None. Sample id of the files to which mutations are added. The ids should match the graphs/counts names, but an equivalence can be set with --sample-name-map.
+- `--germline`: Default = ''. Absolute path of the germline mutation file. Format: VCF, MAF or h5.
+- `--somatic`: Default = ''. Absolute path of the somatic mutation file. Format: VCF, MAF or h5.
+- `--sample-name-map`: Default = None. Name mapping to sample names from graphs/counts files. Format: No header. Two columns: *[name of count/graphs file \t name of mutation/pickle file]*. Three columns: *[name of count/graphs file \t name of germline file \t name of somatic file]*. #TODO: add example?
+- `--use-mut-pickle`: Default = False. Set to True to save and use pickled mutation dictionary.
```

Comment:
Add same details as in the pickle-sample parameter

Reply:
Added in last commit